packages/Webkul/Shop/src/Routes/api.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Support\Facades\Route;
+use Webkul\Shop\Http\Controllers\CartController;
+use Webkul\Shop\Http\Controllers\ProductController;
+use Webkul\Shop\Http\Controllers\Customer\Account\CompareController;
+use Webkul\Shop\Http\Controllers\Customer\WishlistController;
+use Webkul\Shop\Http\Controllers\CategoryController;
+
+Route::group(['middleware' => ['locale', 'theme', 'currency'], 'prefix' => 'api'], function () {
+
+    Route::controller(ProductController::class)->group(function () {
+        Route::get('category-products/{categoryId}', 'getByCategory')
```

Comment:
GET `products` -> ProductController@index

The remaining filters, categories, etc will be handled from query params.

Reply:
Done